docs/themes/thxvscode/assets/scss/includes/_docs.scss

```diff
@@ -191,6 +187,14 @@
 	> li li > a {
 		padding: 5px 25px;
 		position: relative;
+
```

Comment:
Nit: any useful comments we can add to the CSS explaining what / why would be useful for future maintenance.